Library/Homebrew/cmd/cleanup.rb

```diff
@@ -19,14 +19,9 @@ def cleanup
 
   def cleanup_logs
     return unless HOMEBREW_LOGS.directory?
-    prune = ARGV.value "prune"
-    if prune
-      time = Time.now - 60 * 60 * 24 * prune.to_i
-    else
-      time = Time.now - 60 * 60 * 24 * 7 * 2 # two weeks
-    end
+    prune = prune_flag 14 # two weeks unless overridden by argument
```

Comment:
What is the difference between `prune = ARGV.value "prune"` (old code, still there in my patch in a different place) and `prune = prune_flag 14` (new code)? Sorry if I'm being blind, but what is the underlying rule and is it documented anywhere? I also kept the style like that because `brew style` didn't complain.